tests/hostcfgd/test_vectors.py

```diff
@@ -79,6 +80,13 @@
             "timezone": "Europe/Kyiv"
         }
     },
+    "MEMORY_STATISTICS": {
+        "memory_statistics": {
+            "enabled": "true",
+            "retention_time": "15",
+            "sampling_interval": "5"
+    }
```

Comment:
nit: indent

Reply:
@FengPan-Frank , thank you for your feedback! I've adjusted the indentation for better readability. Please let me know if there are any other improvements needed.